src/jdk.hotspot.agent/doc/transported_core.html

```diff
@@ -10,101 +10,87 @@ <h1>Debugging transported core dumps</h1>
+and instead you need to include copies of the core dump machine's libraries
+and java installation on the debugger machine, and you need
+to tell SA where these are all located. That is done differently for each
+OS type, and is described in the following sections.
 </p>
 
-<h3>Solaris SA against transported core dumps</h3>
-
+<h3>Using  transported core dumps on Linux</h3>
 <p>
-With transported core dumps, you may get "rtld_db failures" or
-"libthread_db failures" or SA may just throw some other error
-(hotspot symbol is missing) when opening the core dump. 
-Environment variable <b>LIBSAPROC_DEBUG</b> may be set to any value
-to debug such scenarios. With this env. var set, SA prints many
-messages in standard error which can be useful for further debugging.
-SA on Solaris uses <b>libproc.so</b> library. This library also
-prints debug messages with env. var <b>LIBPROC_DEBUG</b>. But,
-setting LIBSAPROC_DEBUG results in setting LIBPROC_DEBUG as well.
+On Linux, SA parses core and shared library ELF files. But, you
+may still face problems with transported core dumps, because matching shared
+objects may not be in the path(s) specified in core dump file. To
+workaround this, you can define environment variable <b>SA_ALTROOT</b>
+to be the directory where shared libraries are kept. In this directory you should
+copy all the needed JDK libraries. You should also copy any referenced linux
+libraries from the core dump machine, unless they are identical to what are
+installed on the debugger machine.
 </p>
 <p>
-The best possible way to debug a transported core dump is to match the
-debugger machine to that of core dump machine. i.e., have same Kernel
-and libthread patch level between the machines. mdb (Solaris modular
-debugger) may be used to find the Kernel patch level of core dump
-machine and debugger machine may be brought to the same level.
+ You can also use <b>SA_ALTROOT</b> to specify a path mapping. For example, if you
+ set <b>SA_ALTROOT=/altroot</b>, then <b>/altroot</b> will be prepended to any path found
+ in the core file, and also prepended to any subdir with the root part stripped off. For example,
+ when looking up <b>/usr/lib/libfoo.so</b>, SA will try to find <b>/altroot/usr/lib/libfoo.so</b>,
+ <b>/altroot/lib/libfoo.so</b>, and <b>/altroot/libfoo.so</b>.
 </p>
+
+<h3>Using  transported core dumps on Windows</h3>
 <p>
-If the matching machine is "far off" in your network, then
-<ul>
-<li>consider using rlogin and <a href="clhsdb.html">CLHSDB - SA command line HSDB interface</a> or
-<li>use SA remote debugging and debug the core from core machine remotely.
-</ul>
+ If the debugger machine and core dump machine have identical Windows libraries, then you only
+ need to point SA to the location of the JDK bin directory. This is done by making sure
+ the bin directory is included in the <b>PATH</b> environment variable. If the windows
```

Comment:
The location of jvm.dll (e.g. %Java_Home%\bin\server) should be included in the PATH environment variable. And if I remember correctly, the location of 'windbg' should also be added to PATH - this needs to be tested and confirmed though.

Reply:
Ok. So you are saying both the bin and bin\server directories have to be in PATH. I'll update for that. I can't verify the need for windbg. I don't have an environment setup for testing this, but Coleen had indicated it wasn't necessary. Are you setup to try it out?